spec/flags_spec.rb

```diff
@@ -0,0 +1,90 @@
+require 'spec_helper'
+
+RSpec.describe ColorLS::Flags do
+  FIXTURES = 'spec/fixtures'.freeze
+
+  subject { capture_stdout { ColorLS::Flags.new(*args).process } }
+
+  def capture_stdout
+    old = $stdout
+    $stdout = fake = StringIO.new
+    yield
+    fake.string
+  ensure
+    $stdout = old
+  end
+
+  context 'with no flags' do
+    let(:args) { [FIXTURES] }
+
+    it { is_expected.to_not match(/((r|-).*(w|-).*(x|-).*){3}/) } # does not list file info
+    it { is_expected.to_not match(/\.hidden-file/) } # does not list hidden files
+    it { is_expected.to_not match(/Found \d+ contents/) } # does not show a report
+    it { is_expected.to match(/a-file.+symlinks.+z-file/) } # sorts all results alphabetically
+    it { is_expected.to match(/a-file.+symlinks/) } # displays dirs & files
```

Comment:
Lines 23 & 24: Can they be split into individual matches of `a-file`, `symlinks` and `z-file`?

Reply:
Oh good catch, I didn't think to reduce the redundancy here when I moved stuff around.